### ISSUE TITLE
Enable markdown support in quick capture

### ISSUE THREAD

**Issue by tejovanthn** [Roam-Research/issues#172]
**Describe the problem**

I tend to note down things as lists/blocks of text, and in Quick Capture mode, all this formatting is stripped to keep a single line of text.

**Feature implementation**

Either enable markdown for quick capture, like it is with the rest of the site, or let us sync text as we input them.

**Comment by wernervp** [comment]
I agree on this, quick-capture seems to focus on capturing a single block. I can understand that from the "quick-capture" perspective. A way to capture formatted text would be a nice addition, but I guess this would be more usable if a new page is created with a Link in quick capture which is probably a new feature request.
